package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "al-collector-js",
+  "version": "1.0.0",
+  "description": "Alert Logic Collector Common Library",
+  "repository": {},
+  "private": true,
+  "scripts": {
+    "lint": "jshint --exclude \"./node_modules/*\" **/*.js",
+    "test": "JUNIT_REPORT_PATH=./test/report.xml nyc --reporter=cobertura mocha --colors --reporter mocha-jenkins-reporter"
+  },
+  "devDependencies": {
+    "aws-sdk": "*",
+    "aws-sdk-mock": "*",
+    "dotenv": "*",
+    "clone": "*",
+    "jshint": "^2.9.5",
+    "mocha": "^3.5.3",
+    "mocha-jenkins-reporter": "^0.3.10",
+    "nyc": "^11.3.0",
+    "rewire": "^2.5.2",
+    "sinon": "^3.3.0"
+  },
+  "dependencies": {
+    "async": "*",
+    "moment": "^2.19.2",
+    "request": "*",
+    "request-promise-native": "*"
+  },
+  "author": "Alert Logic Inc."
+}
```

Comment:
Have you cleaned up deps? Maybe we don't need all of them (just idea not sure)

Reply:
Yes I cleaned them up.   These are all necessary, I think.